samples/plugins/postgres_jdbc_kerberos/manifest.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<connector-plugin class='postgres_jdbc_kerberos' superclass='jdbc' plugin-version='0.0.0' name='Postgres JDBC with Kerberos' version='18.1' min-version-tableau='2019.4'>
```

Comment:
Do you want to change the min version to 2020.2 if its only going to come in after that release ?